modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/UserCardTag.java

```diff
@@ -0,0 +1,512 @@
+	}
+
+	public void setSubtitle(String subtitle) {
+		_subtitle = subtitle;
+	}
+
+	public void setUserCard(UserCard userCard) {
+		_userCard = userCard;
+	}
+
+	public void setUserColorClass(String userColorClass) {
+		_userColorClass = userColorClass;
+	}
+
+	@Override
+	protected void cleanUp() {
+		super.cleanUp();
+
+		_actionDropdownItems = null;
+		_disabled = false;
+		_groupName = null;
+		_href = null;
+		_icon = null;
+		_imageAlt = null;
+		_imageSrc = null;
+		_inputName = null;
+		_inputValue = null;
+		_labels = null;
+		_name = null;
+		_selectable = false;
+		_selected = false;
+		_spritemap = null;
+		_subtitle = null;
+		_userCard = null;
+		_userColorClass = null;
+	}
+
+	@Override
+	protected String getHydratedModuleName() {
+		return "frontend-taglib-clay/UserCard";
+	}
+
+	@Override
+	protected Map<String, Object> prepareProps(Map<String, Object> props) {
+		props.put("actions", _actionDropdownItems);
+		props.put("description", _subtitle);
+		props.put("disabled", _disabled);
+		props.put("href", _href);
+		props.put("userImageAlt", _imageAlt);
+		props.put("userImageSrc", _imageSrc);
+		//props.put("checkboxProps", {name: inputName, value: inputValue});
+		props.put("labels", _labels);
+		props.put("name", _name);
+		props.put("selectable", _selectable);
+		props.put("selected", _selected);
+		props.put("spritemap", _spritemap);
+		props.put("userDisplayType", _userColorClass);
+
+		return super.prepareProps(props);
+	}
+
+	@Override
+	protected String processCssClasses(Set<String> cssClasses) {
+		cssClasses.add("card-type-asset");
+		cssClasses.add("user-card");
+		cssClasses.add("form-check");
+		cssClasses.add("form-check-card");
+		cssClasses.add("form-check-top-left");
+
+		if (!_selectable) {
+			cssClasses.add("card");
+		}
+
+		return super.processCssClasses(cssClasses);
+	}
+
+	@Override
+	protected int processStartTag() throws Exception {
+		super.processStartTag();
+
+		JspWriter jspWriter = pageContext.getOut();
+
+		jspWriter.write("<div class=\"card\">");
+		jspWriter.write("<div class=\"aspect-ratio card-item-first\">");
+
+		if (_selectable) {
+			jspWriter.write("<div class=\"custom-control custom-checkbox\">");
+			jspWriter.write("<label>");
+			jspWriter.write("<input ");
+
+			if (_disabled) {
+				jspWriter.write("disabled=\"disabled\" ");
+			}
+
+			if (Validator.isNotNull(_inputName)) {
+				jspWriter.write("name=\"" + _inputName + "\" ");
+			}
+
+			if (Validator.isNotNull(_inputValue)) {
+				jspWriter.write("name=\"" + _inputValue + "\" ");
+			}
+
+			if (_selected) {
+				jspWriter.write("checked=\"checked\" ");
+			}
+
+			jspWriter.write("class=\"custom-control-input\" ");
+			jspWriter.write("type=\"checkbox\">");
+			jspWriter.write("<span class=\"custom-control-label\"></span>");
+		}
+
+		jspWriter.write("<div class=\"aspect-ratio-item-center-middle ");
+		jspWriter.write("card-type-asset-icon\">");
+		jspWriter.write("<span class=\"sticker sticker-user-icon ");
```

Comment:
Use `StickerTag`?

Reply:
@jbalsas I just wanted to say that I tried that in my on going PR for `VerticalTag`, `FileCard` and `ImageCard` but it seems that in some cases the markup is different.

Reply:
Maybe one is lying then :)

Reply:
How different?

Reply:
Let me just try again and confirm,  if it's a missing css class I can probably add it via a setter (didn't try that yet)

Reply:
@julien done here https://github.com/liferay-frontend/liferay-portal/pull/434/files#diff-7df0e8ed3761415125433a5b6417c3f72c6b95449afa355c819a7cfccf796daeR422